### DISCUSSION TITLE
B站最新限制，没有实名认证不能看直播

### DISCUSSION THREAD

**Discussion by ghost** [the1812/Bilibili-Evolved#4811] [功能建议 / Ideas]
没有实名认证的用户大部分游戏区主播都不让看了（就很离谱，不是开播，连看都不让了），少部分主播（海外主播和少数国内主播）还能继续看，不确定筛选的条件是什么。

实际上所有直播间的视频流其实是可以正常接收的，但在加载直播间组件的时候如果发现用户没有实名认证就会掐掉视频流并弹出提示“请完善账号实名信息后再尝试观看”，不知道能不能绕过这层检测让视频流不被中断。
![image](https://github.com/the1812/Bilibili-Evolved/assets/115329998/8fcaf6c4-97e8-425b-8e3b-b1a089cd147c)

**Comment by Tinhone** [comment]
这里的实名认证是指手机号认证还是身份证认证？如果是手机号认证也许可以视为 B 站降本的新举措——B 站最近也禁止未登录账号的用户对某些直播间的访问了

![图片](https://github.com/the1812/Bilibili-Evolved/assets/81268244/7cea05b7-350b-4701-b2b5-5242d1e66960)

**Reply by ghost** [reply]
> 这里的实名认证是指手机号认证还是身份证认证？如果是手机号认证也许可以视为 B 站降本的新举措——B 站最近也禁止未登录账号的用户对某些直播间的访问了
> 
> ![图片](https://private-user-images.githubusercontent.com/81268244/345713851-7cea05b7-350b-4701-b2b5-5242d1e66960.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAxMTI0MjIsIm5iZiI6MTcyMDExMjEyMiwicGF0aCI6Ii84MTI2ODI0NC8zNDU3MTM4NTEtN2NlYTA1YjctMzUwYi00NzAxLWIyYjUtNTI0MmQxZTY2OTYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA0VDE2NTUyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYxOTVjZTg4MzlhNWMxZDEyOTQyM2Q4OTc0ZTlhOTI5YTdlMDlhZWNlNWY2MTNiMmU2ZWVmZmI3MjVjM2ExN2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.I-u6o09NoLz8qoWaJuNGSOyzaFSB0GRdG721umXhwIU)

实名认证通常要使用有效身份证件认证，验证手机号不叫实名认证。我目前所用账号使用 Google Voice 注册，未经实名认证，很多主播没法观看，均提示「请完善账号实名信息后再尝试观看」

**Comment by llc0930** [comment]
現在虛擬singer也鎖了...破站藥丸

**Reply by llc0930** [reply]
不對，整個虛擬區都鎖了？虛擬日常也進不了

**Comment by llc0930** [comment]
> 没有实名认证的用户大部分游戏区主播都不让看了（就很离谱，不是开播，连看都不让了），少部分主播（海外主播和少数国内主播）还能继续看，不确定筛选的条件是什么。

條件是虛擬gamer區

**Comment by ghost** [comment]
如今实名认证限制取消，只要登录就能看了

**Comment by llc0930** [comment]
13號就解鎖了啊？不過虛擬gamer區還是鎖實名的

**Reply by ghost** [reply]
> 13號就解鎖了啊？不過虛擬gamer區還是鎖實名的

不清楚 B 站如何限制，我现在进哪个主播直播间都不会受此限制，不知道为什么你还不行

**Reply by ghost** [reply]
> 13號就解鎖了啊？不過虛擬gamer區還是鎖實名的

装上这个脚本试试：https://greasyfork.org/zh-CN/scripts/486358-bilibili%E7%9B%B4%E6%92%AD%E5%85%8D%E7%99%BB%E5%BD%95

我装上这个脚本，没登录账号也能正常观看

**Reply by llc0930** [reply]
先收藏了，謝啦
